app/backend/aquifers/models.py

```diff
@@ -104,10 +99,7 @@ class Meta:
         verbose_name_plural = 'Aquifer Productivity Codes'
 
     def __str__(self):
-        return '%s - %s' % (
-            self.code,
-            self.description
-        )
+        return 'aquifer_productivity_code {} {}'.format(self.code, self.description)
```

Comment:
Users don't see this string but I think it would be best to keep it clean and simple. `self.code` and `self.description` should be sufficient. For other cases where the code and description are basically the same, `self.code` is good enough.

Reply:
fixed..